bucket/tageditor.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "3.4.2",
+    "description": "A tag editor with Qt GUI and command-line interface supporting MP4/M4A/AAC (iTunes), ID3, Vorbis, Opus, FLAC and Matroska.",
+    "homepage": "https://github.com/Martchus/tageditor",
+    "license": "GPL-2.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-x86_64-w64-mingw32.exe.zip",
+            "hash": "ff4725dd63f5b8961dedd051c4b22f635d791f337adc2fc43c576e93c9250a5c"
+        },
+        "32bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-i686-w64-mingw32.exe.zip",
+            "hash": "0163b9f1fd0cd44bb18887fbeecdd66de461113735ea704de23e54e0a4a98e6a"
+        }
+    },
+    "pre_install": "Rename-Item $dir\\tageditor*.exe $dir\\tageditor.exe",
```

Comment:
- Why did you remove the quotes?
- Rename-Item does not take wildcard
- Rename-Item does not take fullpath for -NewName